From 447900d7afdca70913c4a66bcce607bebb1b40c8 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 21 Dec 2016 14:12:58 +0800
Subject: [PATCH 04/13] CHROMIUM: arm64: dts: rockchip: change the dmc opp
 table for bob

The bob will select 800M as the dram max frequency for bob boards,
Since the 928M ddr frequency will cause the random crash on bob board,
that should be some reason limited by hardware. We can add it if bob
want to support the 928M in the future.

BUG=chrome-os-partner:61001
TEST=cat /sys/devices/platform/dmc/devfreq/dmc/available_frequencies

Change-Id: Ib3137e70a6e0003672d43ccb3ded9efe1995fec7
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Reviewed-on: https://chromium-review.googlesource.com/422969
Reviewed-by: Brian Norris <briannorris@chromium.org>
---
 arch/arm64/boot/dts/rockchip/rk3399-gru-bob.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-gru-bob.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-gru-bob.dtsi
index 429644db42cb..4fea1d6fc810 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-gru-bob.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru-bob.dtsi
@@ -100,6 +100,14 @@
 	};
 };
 
+&dmc_opp_table {
+	opp02 {
+		opp-suspend;
+	};
+
+	/delete-node/ opp03;
+};
+
 &edp {
 	status = "okay";
 
-- 
2.17.0

